packages/volto/news/6493.bugfix

```diff
@@ -0,0 +1 @@
+Fixed body class handling to prevent query string inclusion in is-adding-contenttype.@Abhishek-17h
```

Comment:
A change log entry should say what the problem was and what was fixed. We publish some guidance at https://6.docs.plone.org/contributing/index.html#write-a-good-change-log-entry.

```suggestion
No longer add classes from the query string search text in the `body` tag. @Abhishek-17h
```